flixel/FlxG.hx

```diff
@@ -558,11 +562,15 @@ class FlxG
 	 */
 	public static inline function openURL(URL:String, Target:String = "_blank"):Void
 	{
+		#if linux
+		Sys.command('/usr/bin/xdg-open', [URL, "&"]);
+		#else
```

Comment:
First, the linux url issue is an issue with openFL and should likely be fixed there. Second, this wouldn't work with `FlxG.open("haxe.org")` which does work in html5 because flixel prepends a "https://" to links without it. I had linux users test this and it worked for all but one, due to their environment setup.

Reply:
```
❯ xdg-open haxe.org
xdg-open: file 'haxe.org' does not exist
```

Reply:
done